### PR TITLE
Setup Continuous Integration

### LINKED ISSUE #165: Setup Continuous Integration

### PR DESCRIPTION
**Pull request by yogsototh**
It support all integration tests with Elasticsearch too.

The build fail actually, but apparently this is a bug waiting to be fixed by a PR (#162).

An admin of `threatgrid` should register an account to travis-ci and authorize the repository.

Closes #165

### CODE REVIEW

**Comment by yogsototh** [comment]
You can see the result of the build for my fork here:

https://travis-ci.org/yogsototh/ctia/builds/120598827

File: resources/ctia-default.properties.ci [hunk 1]
```diff
@@ -0,0 +1,10 @@
+auth.service=allow-all
```
Comment (polygloton) [inline resources/ctia-default.properties.ci]: Is it possible to put this in the test/resources directory?

**Comment by polygloton** [comment]
Looks good except for my one comment.

**Comment by polygloton** [comment]
Cool!

**Comment by gbuisson** [comment]
looks good !

**Pull request merged**